crates/sncast/tests/e2e/declare.rs

```diff
@@ -244,6 +244,7 @@ async fn test_contract_already_declared() {
         accounts_json_path.as_str(),
         "--account",
         "user1",
+        "--wait",
```

Comment:
Why is this added to an existing test?

Reply:
Are you sure? I removed `--wait` from this test, run all `delcare` tests and they pass so maybe we don't have to add it 🤔 ?